src/extras/crosshair.js

```diff
@@ -28,11 +28,19 @@ Dygraph.Plugins.Crosshair = (function _extras_crosshair_closure() {
 
   var crosshair = function crosshair(opt_options) {
     this.canvas_ = document.createElement("canvas");
+    this.updateCanvasSize(0, 0)
     opt_options = opt_options || {};
     this.direction_ = opt_options.direction || null;
     this.strokeStyle_ = opt_options.strokeStyle || "rgba(0, 0, 0, 0.3)";
   };
 
+  crosshair.prototype.updateCanvasSize = function updateCanvasSize(width, height) {
+    this.canvas_.width = width
+    this.canvas_.height = height
+    this.canvas_.style.width = width.toString(10) + 'px'    // for IE
```

Comment:
10 is default

Reply:
and the `toString` is pretty redundant, the entire rest of the code doesn’t use it either, so unless you can provide a very good reason, please revert to the string concatenation

Reply:
Confused by eslint [radix rule](https://eslint.org/docs/latest/rules/radix). Using 10 as optimization to skip possible checks inside. No need on object that is a number for sure